wordpress-mobile/WordPress-Android
# Add Remove functionality to the quick start dynamic cards

Fixes #13852

This PR adds remove functionality to the QS dynamic cards. The removal is permanent and there is no going back except reinstalling the app (or clearing cache). The data is kept in the App preferences.

To test:
- Make sure the my site improvements flag is turned on and the app is restarted
- Make sure your site has QuickStart started
- Click on card/hamburger menu/don't show again
- Notice the card disappears
- Restart the app
- Notice the card is still gone


https://user-images.githubusercontent.com/1079756/106914093-f69dd100-6704-11eb-9156-419ecf367a15.mp4

PR submission checklist:
